internal/db/migrations/004_pubkeys.up.sql

```diff
@@ -8,4 +8,21 @@ CREATE TABLE pubkeys
   body TEXT NOT NULL CHECK (NOT empty(body))
 );
 
+CREATE TABLE pubkey_resources
+(
+  id BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+  tag TEXT NOT NULL DEFAULT random_string(15),
+  pubkey_id BIGINT NOT NULL REFERENCES pubkeys(id),
+  provider INTEGER NOT NULL CHECK (valid_provider(provider)),
+  handle TEXT NOT NULL
+);
```

Comment:
Column `tag` is a random string that is used as a handler in the future. I considered UUID but I think it is too long and users might be bothered with such a long tag. The tag does not need to be unique, its just something that is hard to guess.

Reply:
Isn't the usefulness of such a tag lower if uniqueness at least per CU account is not guaranteed?
If I can't reliably verify that this key is or is not present in the cloud provider using this `tag`, I do not see much value of it.

Reply:
Answered below.